themes/orbit/layout/partials/code_sample.ejs

```diff
@@ -0,0 +1,8 @@
+<pre>
```

Comment:
This may seem a bit silly since there's already and auto-syntax highlighter in Hexo, but I really wanted it <fuzzy-designer-hat>just so</fussy-designer-hat>.